### PR TITLE
Map Examples Responsive Fix (#7420)

### LINKED ISSUE #7420: Examples are not mobile-friendly
- [x] https://leafletjs.com/examples/layers-control/example.html
Layers control is out of view on mobile.
  (Edit: Fixed in https://github.com/Leaflet/Leaflet/pull/7950)
- [x] Todo: check the rest of examples
  (Edit: Fixed in https://github.com/Leaflet/Leaflet/pull/7950)
- [ ] [Responsive media elements](https://github.com/Leaflet/Leaflet/issues/7420#issuecomment-1023022670)

_(Edited by @Malvoz)_

### PR DESCRIPTION
**Pull request by HRSArgyropoulos**
fixes #7420 where maps were expanding outside the container.
Had to remove fixed values in cloropleth and quick-start example.
Examples tested on localhost (ruby/jekyll/bundler).
Please let me if/what you do not like so i can come back to this if needed.

### CODE REVIEW

**Comment by johnd0e** [comment]
Using term "Responsive Fix" do you mean that samples now adapts to screen width? Or anything else?

**Comment by HRSArgyropoulos** [comment]
> Using term "Responsive Fix" do you mean that samples now adapts to screen width? Or anything else?

Yes, if you take a look at the examples now on the website (tutorials) it forces the page to be scrollable. So we restrain the maps with a max-width and give them 100% width of the container. And also for the maps to not be scrollable we also need to change the iframe width. The height will remain as is.

p.s. the pov/zoom height should probably taken into consideration here as the map size becomes smaller.

File: docs/_layouts/tutorial_frame.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 			margin: 0;
 		}
 		#map {
-			width: 600px;
+			width: 100%;
```
Comment (Malvoz) [inline docs/_layouts/tutorial_frame.html:22]: If an initial width of 600px is desired (keeping `width: 600px` as currently set) then `max-width: 100%` could be used additionally to accomplish responsiveness of the maps.

File: docs/examples/layers-control/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ title: Layer Groups and Layers Control
 
 This tutorial will show you how to group several layers into one, and how to use the layers control to allow users to easily switch different layers on your map.
 
-{% include frame.html url="example.html" %}
+{% include frame.html width="100%" height="400" url="example.html" %}
```
Comment (Malvoz) [inline docs/examples/layers-control/index.md:10]: In HTML it's invalid to use % in `width` and `height` attributes, although some browsers seem to respect it. Just an FYI.

File: docs/_layouts/tutorial_frame.html (hunk 1 above)
Comment (HRSArgyropoulos) [reply docs/_layouts/tutorial_frame.html:22]: The default width was 600px so i thought we wanted to keep it that way and from then making it smaller. I changed the width to your proposal, the desired result is still the same so its cool :)

File: docs/examples/layers-control/index.md (hunk 2 above)
Comment (HRSArgyropoulos) [reply docs/examples/layers-control/index.md:10]: The was left from a test that i was doing with setting value to attributes of the map. Thanks for letting me know. Removed them and the example is working as the rest.

**Comment by johnd0e** [comment]
@HRSArgyropoulos 
Please resolve conflicts

**Comment by Falke-Design** [comment]
@Malvoz is this PR completly superseded by #7950? (Ignore the `mapid` changes)

**Comment by Malvoz** [comment]
@Falke-Design Yes, I think https://github.com/Leaflet/Leaflet/issues/7420 involves a bit more than just responsive _maps_, so keeping that open however. 👍🏼

**Pull request closed without merging**